docs/data/profile-properties.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Profile Properties 
+description: Profile Properties allow you to sync a current property value from a source (currently only Snowflake) to use within the Amplitude platform.
+---
+
+## Context
+Customers can now bring in non-behavioral, customer profile data from their data warehouse to merge it with the existing behavioral product data already living in Amplitude. These values will always display the most current value being synced from the customer’s data warehouse.
+
+## Setup
+To set up a Profile Property in Amplitude, you must connect Amplitude to your data warehouse. Once connected, and you reach the data configuration section, you must select the “Warehouse Props” data type in the dropdown. From there, there are two minimum requirements for the import: a user identifier (user_id) and a profile property. Note: you may add more than one warehouse property per import, but there must be at least one per import.
```

Comment:
Need some additional context here, and we should format this in an ordered list.